contractcourt/chain_watcher.go

```diff
@@ -598,6 +607,13 @@ func (c *chainWatcher) dispatchContractBreach(spendEvent *chainntnfs.SpendDetail
 			return spew.Sdump(retribution)
 		}))
 
+	// Hand the retribution info over to the breach arbiter.
+	if err := c.contractBreach(retribution); err != nil {
```

Comment:
This and `contractClose` are great, I like that the use of channels is hidden behind a function call 👍